README.Rmd

```diff
@@ -118,7 +118,10 @@ explanation <- explain(
   prediction_zero = p
 )
 
-# Printing the Shapley values for the test data
+# Printing the Shapley values for the test data.
```

Comment:
I think we should avoid commenting this much, and rather refer to `?shapr::explain`.

I.e. you could do 
```
# Printing the Shapley values for the test data. For more information about the interpretation of the values see ?shapr::explain.  
```

Reply:
Agree. Fixed with 5e8371ce6917e28ab0909336bc3a9dc054d8d8b8